src/0_utils.js

```diff
@@ -74,20 +74,6 @@ utils.hashValue = function(key) {
 		return location.hash.match(new RegExp(key + ':([^&]*)'))[1];
 	}
 	catch (e) {
-		return '';
-	}
-};
-
-// TODO: needs a better name
-utils.mobileReady = function() {
-	return navigator.userAgent.match(/android|i(os|p(hone|od|ad))/i);
-};
-
-// TODO: this probably should not be a utility
-utils.closeBanner = function() {
-	var d = document.getElementById('branch-banner');
-	if (d) {
-		d.parentNode.removeChild(d);
-		document.body.style.marginTop = '0px';
+		return ''; //TODO: Look into why we dont return the error
```

Comment:
I don't know if this would really even throw an error :P this is just code adopted from before